.eslintrc

```diff
@@ -20,16 +20,21 @@
         "project": ["./tsconfig.json"],
         "tsconfigRootDir": "."
     },
+    "ignorePatterns": [".eslintrc"],
     "rules": {
         "semi": "error",
         "no-extra-semi": "error",
         "no-eval": "error",
-        "@typescript-eslint/prefer-readonly": "error",
         // TODO review each exclusion and fix issues if needed
+        "@typescript-eslint/explicit-module-boundary-types": "off",
+        "@typescript-eslint/no-unsafe-return": "off",
+        "@typescript-eslint/no-unsafe-call": "off",
+        "@typescript-eslint/no-unsafe-assignment": "off",
+        "@typescript-eslint/no-unsafe-member-access": "off",
         "@typescript-eslint/explicit-function-return-type": "off",
+        "@typescript-eslint/restrict-template-expressions": "off",
         "@typescript-eslint/no-explicit-any": "off",
         "@typescript-eslint/no-non-null-assertion": "off",
-        "@typescript-eslint/no-unused-vars": "off",
         "@typescript-eslint/no-use-before-define": "off",
         "@typescript-eslint/require-await": "off",
```

Comment:
maybe it is worth to adjust these useful rules ?

Reply:
Done